ge04_team01/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    is_new_customer = fields.Boolean(related="partner_id.is_new_customer")
```

Comment:
I don't think you need this field.

Reply:
I think it's necessary because otherwise we can't add it into the Sale Order form viev.

Reply:
Sorry, did not realize you need the field on the view